addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -56,9 +56,11 @@
             <field name="arch" type="xml">
                 <form string="Mailing">
                     <header style="min-height:31px;">
-                        <button name="action_put_in_queue" type="object" attrs="{'invisible': [('state', 'in', ('in_queue', 'done'))]}" class="oe_highlight" string="Send"
+                        <button id="action_launch_now" name="action_put_in_queue" type="object" class="oe_highlight" string="Launch"
```

Comment:
id="action_launch_now" name="action_put_in_queue"

Mmmh, launch now or put in queue ? What is the purpose of the id ?

With action_launch_scheduled button herebelow, seems like we could really improve a bit the naming / IDs of all that brol.